roles/pulibrary.patroni/defaults/main.yml

```diff
@@ -0,0 +1,189 @@
+---
+# defaults file for roles/pulibrary.patroni
+patroni_postgresql_version: 11
+patroni_postgresql_exists: false
+
+patroni_config_dir: /etc/patroni
+
+patroni_pip_packages:
+  - { name: "setuptools", state: "latest", umask: "0022" }
+  - { name: "patroni[{{ patroni_dcs }}]", state: "present", umask: "0022" }
+
+patroni_dcs: consul
+patroni_dcs_exists: false
+
+patroni_exec_start_pre: "/bin/mkdir -m 2750 -p /var/run/postgresql/{{ patroni_postgresql_version }}-main.pg_stat_tmp"
+
+patroni_replication_username: replicator
+patroni_replication_password: repuserpasswd
+patroni_superuser_username: postgres
+patroni_superuser_password: supersecretpostgrespasswd
```

Comment:
Should these values be imported from Vault values or are they overridden later?

Reply:
kayiwa directed me to https://github.com/pulibrary/princeton_ansible/blob/577_patroni_postgres_role/group_vars/patroni/staging.yml, where it is indeed set.

Reply:
overridden by `group_vars/patroni/` variables (I made an issue) however. https://github.com/pulibrary/princeton_ansible/issues/872